scripts/normalize-by-median.py

```diff
@@ -233,13 +236,20 @@ def main():  # pylint: disable=too-many-branches,too-many-statements
     discarded = 0
     input_filename = None
 
-    for index, input_filename in enumerate(args.input_filenames):
-        if args.single_output_filename != '':
-            output_name = args.single_output_filename
+    auto_output = False
```

Comment:
The addition of auto_output is a bit confusing and, I think, unnecessary.  What about removing it, and (inside the 'for' loop) putting 'if not args.single_output_filename'? This seems like it would be simpler to me.